<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="3" passed="0" failed="0" skipped="3">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-02-19T05:03:24Z" name="Suite" finished-at="2019-02-19T05:03:24Z" duration-ms="354">
    <groups>
    </groups>
    <test started-at="2019-02-19T05:03:24Z" name="Test" finished-at="2019-02-19T05:03:24Z" duration-ms="354">
      <class name="com.abc.testscript.TC002MapSearchPage">
        <test-method is-config="true" signature="Beforem()[pri:0, instance:com.abc.testscript.TC002MapSearchPage@5f8edcc5]" started-at="2019-02-19T10:33:24Z" name="Beforem" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Beforem -->
        <test-method is-config="true" signature="Beforem()[pri:0, instance:com.abc.testscript.TC002MapSearchPage@5f8edcc5]" started-at="2019-02-19T10:33:24Z" name="Beforem" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Beforem -->
        <test-method signature="SearchPageTest(com.abc.dataprovider.ExcelDataProviderObject)[pri:0, instance:com.abc.testscript.TC002MapSearchPage@5f8edcc5]" started-at="2019-02-19T10:33:24Z" name="SearchPageTest" data-provider="getGoogleMapsData" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.abc.dataprovider.ExcelDataProviderObject@6aa61224]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SearchPageTest -->
        <test-method is-config="true" signature="AfterTest()[pri:0, instance:com.abc.testscript.TC002MapSearchPage@5f8edcc5]" started-at="2019-02-19T10:33:24Z" name="AfterTest" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.abc.projectpage.TestngAnnotations.AfterTest(TestngAnnotations.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTest -->
        <test-method signature="SearchPageTest(com.abc.dataprovider.ExcelDataProviderObject)[pri:0, instance:com.abc.testscript.TC002MapSearchPage@5f8edcc5]" started-at="2019-02-19T10:33:24Z" name="SearchPageTest" data-provider="getGoogleMapsData" finished-at="2019-02-19T10:33:24Z" duration-ms="16" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.abc.dataprovider.ExcelDataProviderObject@7357a011]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SearchPageTest -->
        <test-method is-config="true" signature="AfterTest()[pri:0, instance:com.abc.testscript.TC002MapSearchPage@5f8edcc5]" started-at="2019-02-19T10:33:24Z" name="AfterTest" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.abc.projectpage.TestngAnnotations.AfterTest(TestngAnnotations.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTest -->
      </class> <!-- com.abc.testscript.TC002MapSearchPage -->
      <class name="com.abc.testscript.TC001ValidateFormTestCase">
        <test-method is-config="true" signature="Beforem()[pri:0, instance:com.abc.testscript.TC001ValidateFormTestCase@10aa41f2]" started-at="2019-02-19T10:33:24Z" name="Beforem" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.abc.seleniumrepository.SeleniumRepo.delete(SeleniumRepo.java:196)
	at com.abc.seleniumrepository.SeleniumRepo.delete(SeleniumRepo.java:205)
	at com.abc.seleniumrepository.SeleniumRepo.deleteTempFile(SeleniumRepo.java:182)
	at com.abc.seleniumrepository.SeleniumRepo.startBrowser(SeleniumRepo.java:50)
	at com.abc.projectpage.TestngAnnotations.Beforem(TestngAnnotations.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Beforem -->
        <test-method is-config="true" signature="AfterTest()[pri:0, instance:com.abc.testscript.TC001ValidateFormTestCase@10aa41f2]" started-at="2019-02-19T10:33:24Z" name="AfterTest" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTest -->
        <test-method is-config="true" signature="BeforeTest()[pri:0, instance:com.abc.testscript.TC001ValidateFormTestCase@10aa41f2]" started-at="2019-02-19T10:33:24Z" name="BeforeTest" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTest -->
        <test-method signature="FormValidationTest()[pri:0, instance:com.abc.testscript.TC001ValidateFormTestCase@10aa41f2]" started-at="2019-02-19T10:33:24Z" name="FormValidationTest" finished-at="2019-02-19T10:33:24Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FormValidationTest -->
      </class> <!-- com.abc.testscript.TC001ValidateFormTestCase -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
